shared/menubar/files.desktop.js

```diff
@@ -135,7 +136,16 @@ const UserTlfUpdateRow = (props: UserTlfUpdateRowProps) => (
           in&nbsp;
         </Kb.Text>
         <Kb.Text type="BodySmallSecondaryLink" style={styles.tlfParticipants} onClick={props.onSelectPath}>
-          {props.tlfType === 'team' ? props.teamname : props.participants.join(',')}
+          {props.tlfType === 'team' ? (
+            props.teamname
+          ) : props.tlfType === 'public' ? (
+            <Kb.Box2 direction="horizontal" fullWidth={true}>
+              {props.participants.join(',')}&nbsp;&nbsp;
```

Comment:
Instead of leaving spaces here, would it possible to make `Meta` take a `style` that has a margin (or wrap it in a `Box` if it doesn't take `style`)?

Reply:
I think something like `gap="xtiny"` for `Box2` would also work.